envoy-filters/mgw-source/filters/http/mgw-wasm-websocket/filter.cc

```diff
@@ -141,7 +141,6 @@ FilterHeadersStatus MgwWebSocketContext::onResponseHeaders(uint32_t, bool) {
   for (auto& p : pairs) {
     if (std::string(p.first) == STATUS_HEADER && std::string(p.second) == STATUS_101) {
       std::string upstream_address;
-      auto buffer = getValue({"upstream", "address"}, &upstream_address);
```

Comment:
It uses `proxy_get_property` ABI function to get the value of upstream address from the stream info of a particular request. Removed it because we are no longer using that value even-though we assign it to the variable `buffer` there.

Reply:
Sorry my bad, we are using the upstream address (passing the upstream address variable reference to getValue()), but not using the returned value of `getValue()`. It returns true/ false depending on the passed key exists or not. Since we are not using it anywhere, `[-Werror,-Wunused-variable]` error occurs during compilation. So added the `getValue()` back without assigning the returned value to a variable. 

getValue() method implementation: https://github.com/proxy-wasm/proxy-wasm-cpp-sdk/blob/c32d380ca6c9b1afac38a3841be99c37af2698bf/proxy_wasm_api.h#L627